### PR TITLE
standardize tense in trade process wording

### PR DESCRIPTION
**Pull request by tylerchambers**
Adjust the wording in the trade process to all have a matching tense.

### CODE REVIEW

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -564,9 +564,9 @@ portfolio.pending.invalidDelayedPayoutTx=Please do NOT send the Altcoin or Fiat
 portfolio.pending.step1.waitForConf=Wait for blockchain confirmation
 portfolio.pending.step2_buyer.startPayment=Start payment
 portfolio.pending.step2_seller.waitPaymentStarted=Wait until payment has started
-portfolio.pending.step3_buyer.waitPaymentArrived=Wait until payment arrived
+portfolio.pending.step3_buyer.waitPaymentArrived=Wait for payment to arrive
 portfolio.pending.step3_seller.confirmPaymentReceived=Confirm payment received
-portfolio.pending.step5.completed=Completed
+portfolio.pending.step5.completed=Complete
```
Comment (ghost) [inline core/src/main/resources/i18n/displayStrings.properties:569]: I suggest to align the key `portfolio.pending.step5.completed` to match the english wording -> `portfolio.pending.step5.complete`

Comment (ghost) [reply core/src/main/resources/i18n/displayStrings.properties:569]: (in both cases)

Comment (ghost) [reply core/src/main/resources/i18n/displayStrings.properties:569]: Um.. @petrhejna I suggest leaving the code tag `portfolio.pending.step5.completed` alone.  Its hardly any different from the proposed text and the ripple effects in the translation files would be quite large.

**Comment by ghost** [comment]
NACK.  I can see that the tense does not match, but really don't think the original wording is so bad it justifies a change.

**Comment by sqrrm** [comment]
I cannot really see the big difference in these formulations. Not worth the pain to change.

**Pull request closed without merging**